### DISCUSSION TITLE
Pterodactyl Deployment Issue

### DISCUSSION THREAD

**Discussion by itsfranciscooli** [eritislami/evobot#1357] [General]
I am trying to deploy Evo Bot in pterodactyl wings. I am using a Basic Discord Bot Hosting Node JS. I've changed the default start up command with **npm run prod** as suggested by someone in a closed issue. It still hasn't fixed the issue.

This is a copy of what i am seeing in the logs.
[https://paste.xmhosts.com/ahidozijev.sql](https://paste.xmhosts.com/ahidozijev.sql)

How can i fix this ? I would really appreciate some help.

**Comment by eritislami** [comment]
```
> npm run build && node ./dist/index.js
> evobot@2.5.2 build
> tsc --build
sh: 1: tsc: Permission denied
```

It's a permission problem, I have 0 experience with whatever provider you are using so I cannot really say how to fix it with them.
